src/pudl_rmi/connect_ferc1_to_eia.py

```diff
@@ -90,123 +94,37 @@ def execute(pudl_out, plant_parts_eia):
 class InputManager:
     """Class prepare inputs for linking FERC1 and EIA."""
 
-    def __init__(self, file_path_training, pudl_out, plant_parts_eia):
+    def __init__(self, train_df, pudl_out, plant_parts_eia_distinct):
         """
         Initialize inputs manager that gets inputs for linking FERC and EIA.
 
         Args:
-            file_path_training (path-like): path to the CSV of training data.
-                The training data needs to have at least two columns:
-                record_id_eia record_id_ferc1.
             pudl_out (object): instance of `pudl.output.pudltabl.PudlTabl()`.
             plant_parts_eia (pandas.DataFrame)
         """
-        self.file_path_training = file_path_training
+        self.train_df = train_df
         self.pudl_out = pudl_out
-        self.plant_parts_eia = plant_parts_eia
+        self.plant_parts_true_df = plant_parts_eia_distinct
 
         # generate empty versions of the inputs.. this let's this class check
         # whether or not the compiled inputs exist before compilnig
```

Comment:
I changed around these inputs so that the full PPL and training data filepath isn't needed. Let me know if I'm misunderstanding something about how these inputs work or how `InputManager` should be structured.